app/views/layouts/admin/_admin_navigation.html.erb

```diff
@@ -30,8 +30,11 @@
 <p class="small text-info ml-3">
 <i class="fa fa-bug"></i> <%= link_to "Report a bug", "https://github.com/geeksforsocialchange/PlaceCal/issues/new?assignees=&labels=&template=bug_report.md" %>
 
+<% tag = `curl  "https://api.github.com/repos/geeksforsocialchange/PlaceCal/tags" | jq -r '.[0].name'` %>
```

Comment:
oh golly.

this is a bit of a security/performance nightmare here. it is not good-practice to run shell scripts like this.
1. curling from a URL may fail
2. the URL can return anything, which we are then piping into `jq` which may have a bug and execute malicious code, possibly exfiltrating sensitive info from our production servers to anywhere
3. what happens if these commands fail? will it bring down production for everyone?
4. do we have jq installed on staging / production?
5. this can be done in ruby, once, and cached